core/controllers/release_coordinator.py

```diff
@@ -58,6 +59,148 @@ def delete(self) -> None:
+):
+    """Handler for user groups."""
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS: Dict[str, str] = {}
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {},
+        'PUT': {
+            'user_group_id': {
+                'schema': {
+                    'type': 'basestring'
+                },
+                'default_value': None
+            },
+            'user_group_name': {
+                'schema': {
+                    'type': 'basestring'
+                },
+                'default_value': None
+            },
+            'user_group_user_usernames': {
+                'schema': {
+                    'type': 'list',
+                    'items': {
+                        'type': 'basestring'
+                    }
+                },
+                'default_value': None
+            }
+        },
+        'DELETE': {
+            'user_group_id': {
+                'schema': {
+                    'type': 'basestring'
+                },
+                'default_value': None
+            }
+        },
+        'POST': {
+            'user_group_name': {
+                'schema': {
+                    'type': 'basestring'
+                },
+                'default_value': None
+            },
+            'user_group_user_usernames': {
+                'schema': {
+                    'type': 'list',
+                    'items': {
+                        'type': 'basestring'
+                    }
+                },
+                'default_value': None
+            }
+        }
+    }
+
+    @acl_decorators.can_access_release_coordinator_page
+    def get(self) -> None:
+        """Populates the data for user groups."""
+        user_groups = user_services.get_all_user_groups()
+        user_groups_dict_list = []
+        for user_group in user_groups:
+            user_groups_dict_list.append(user_group.to_dict())
+
+        self.render_json({
+            'user_group_dicts': user_groups_dict_list
+        })
+
+    @acl_decorators.can_access_release_coordinator_page
+    def delete(self) -> None:
+        """Performs deletion on the specified user group."""
+        assert self.user_id is not None
+        assert self.normalized_request is not None
+        try:
+            user_group_id = self.normalized_request['user_group_id']
+            assert user_group_id is not None
+            user_services.delete_user_group(user_group_id)
+            self.render_json(self.values)
+        except Exception as e:
+            logging.exception('[RELEASE-COORDINATOR] %s', e)
+            self.render_json({'error': str(e)})
+            raise e
+
+    @acl_decorators.can_access_release_coordinator_page
+    def put(self) -> None:
+        """Updates the specified user group."""
+        assert self.user_id is not None
+        assert self.normalized_payload is not None
+        try:
+            user_group_name = (
```

Comment:
Why do you have a default_value of None in the handler schema if we don't accept None values for the update? Should those default values be dropped?

Reply:
Right, I have updated this.